brigade-worker/src/index.ts

```diff
@@ -28,15 +28,57 @@
 import * as fs from "fs";
 import * as process from "process";
 import * as ulid from "ulid";
+import * as moduleAlias from "module-alias";
 
 import * as events from "@brigadecore/brigadier/out/events";
 import { App } from "./app";
 import { ContextLogger, LogLevel } from "@brigadecore/brigadier/out/logger";
 
 import { options } from "./k8s";
 
-// This is a side-effect import.
-import "./brigade";
+// Install aliases for common ways of referring to Brigade/Brigadier.
+moduleAlias.addAliases({
+  "brigade": __dirname + "/brigadier",
+  "brigadier": __dirname + "/brigadier",
+  "@brigadecore/brigadier": __dirname + "/brigadier",
+});
+// Add the current module resolution paths to module-alias, so the node_modules
+// that prestart.js adds to will be resolvable from the Brigade script and any
+// local dependencies.
+module.paths.forEach(moduleAlias.addPath);
+moduleAlias();
+
+// Script locations in order of precedence.
```

Comment:
Maybe this whole block (through the `findScript` function) would be better as a separate module? It feels a little distracting to me inline like this.